homeassistant/components/tuya/light.py

```diff
@@ -526,6 +528,19 @@ def __init__(
         ):
             self._color_temp = int_type
             color_modes.add(ColorMode.COLOR_TEMP)
+            self._white_color_mode = ColorMode.COLOR_TEMP
+        elif (
+            workmode_type
```

Comment:
We read this above without specifying the dptype and store in `self._color_mode_dpcode`
Now we read it with dptype set to `DPType.ENUM` and store in a variable named `workmode_type`.

Maybe it would be clearer to store the result in a variable named `color_mode_dpcode_enum`?

Reply:
Local variable renamed.

As discussed on Discord, we should merge `_color_mode_dpcode` and the new variable, but that should be done in a follow-up PR.